php/class-coauthors-wp-list-table.php

```diff
@@ -97,14 +99,35 @@ function prepare_items() {
 		switch ( $this->active_filter ) {
 			case 'with-linked-account':
 			case 'without-linked-account':
-				$args['meta_key'] = $coauthors_plus->guest_authors->get_post_meta_key( 'linked_account' );
+				$key = $coauthors_plus->guest_authors->get_post_meta_key( 'linked_account' );
 				if ( 'with-linked-account' == $this->active_filter ) {
-					$args['meta_compare'] = '!=';
+                    $args['meta_query'] = array(
```

Comment:
Looks like this does not get advantages of `switch case`. Instead of using this condition `if ( 'with-linked-account' == $this->active_filter )`, what's about moving the value of `$args['meta_query']` to each `case`?

Reply:
> Looks like this does not get advantages of switch case. Instead of using this condition if ( 'with-linked-account' == $this->active_filter ), what's about moving the value of $args['meta_query'] to each case?

Absolutely. I was so focused on the query, that I didn't notice the strange switch case statement. I just updated that section and used a compact version to increase the readability. At least, I think the code is more readable with the compact version. Let me know what you think about it.

Reply:
LGTM 👍